pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -434,6 +436,10 @@ func hashMeta(m metav1.ObjectMeta) string {
 	})
 }
 
+func hashData(data map[string][]byte) string {
```

Comment:
I don't think we need a wrapper for `utils.ObjectHash` should just call `utils.ObjectHash` directly 🙂